main_comment_test.go

```diff
@@ -250,3 +285,64 @@ func TestProcessGitHubWebhook(t *testing.T) {
 		})
 	}
 }
+
+func TestParsePrOptions(t *testing.T) {
+	testCases := map[string]struct {
+		StartPipelineComment string
+		RepoToPr             map[string]string
+	}{
+		"start pipeline with --pr flags": {
+			StartPipelineComment: "start pipeline --pr mender-connect/pull/88/head --pr deviceconnect/pull/12/head --pr mender/3.1.x",
+			RepoToPr: map[string]string{
+				"mender-connect": "pull/88/head",
+				"deviceconnect":  "pull/12/head",
+				"mender":         "3.1.x",
+			},
+		},
+		"start pipeline with --pr flags and some sugar": {
+			StartPipelineComment: "start pipeline --pr mender-connect/pull/88/head --pr deviceconnect/pull/12/head --pr mender/3.1.x sugar pretty please",
+			RepoToPr: map[string]string{
+				"mender-connect": "pull/88/head",
+				"deviceconnect":  "pull/12/head",
+				"mender":         "3.1.x",
+			},
+		},
+		"start pipeline with --pr flags and some sugar with multiple spaces": {
+			StartPipelineComment: "start pipeline  --pr          mender-connect/pull/88/head          --pr          deviceconnect/pull/12/head --pr mender/3. 1.x     sugar pretty please",
+			RepoToPr: map[string]string{
+				"mender-connect": "pull/88/head",
+				"deviceconnect":  "pull/12/head",
+				"mender":         "3.",
```

Comment:
Question, what would happen in this instance?
Would the bot post some kind of error message?

Reply:
I thought about checking if the revision exists -- I can add that later, as this can unblock Maciek, perhaps the simpler parse error handler will be enough for the first iteration? (I added, see above)